### PR TITLE
Api void

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
original PR: https://github.com/demisto/content/pull/6401
contribution PR: https://github.com/demisto/content/pull/6000

## Description
new integration with "APIVoid".

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Packs/APIVoid/Integrations/APIVoid/Pipfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+idna = "==2.8"
+olefile = "==0.46"
+python-dateutil = "==2.8.0"
+pytz = "==2019.1"
+regex = "==2019.6.8"
+requests = "==2.22.0"
+six = "==1.12.0"
+tzlocal = "==1.5.1"
+urllib3 = "==1.25.3"
+PyYAML = "==5.1.1"
```
Comment (ronykoz) [inline Packs/APIVoid/Integrations/APIVoid/Pipfile:30]: do we need all of them?

File: Packs/APIVoid/Integrations/APIVoid/APIVoid.py [hunk 2]
```diff
@@ -0,0 +1,621 @@
...
+    Client will implement the service API, and should not contain any Demisto logic.
+    Should only do requests and return data.
+    """
+    def __init__(self, base_url, apikey, verify, proxy, good, suspicious, bad, malicious):
+        self.apikey = apikey
+        super().__init__(base_url, verify=verify, proxy=proxy)
+        self.good = good
+        self.suspicious = suspicious
+        self.bad = bad
+        self.malicious = malicious
```
Comment (ronykoz) [inline Packs/APIVoid/Integrations/APIVoid/APIVoid.py:29]: why is it here?

File: Packs/APIVoid/Integrations/APIVoid/APIVoid.py [hunk 3]
```diff
@@ -0,0 +1,621 @@
...
+    raw_response = client.check_url(url)
+
+    if 'error' in raw_response:
+        return_error("Command Failed: " + str(raw_response['error']))
+
+    report = raw_response.get('data', {}).get('report', None)
+    if report:
+        report['url'] = url
+
+        engines = len(report.get('domain_blacklist', {}).get('engines', 0))
```
Comment (ronykoz) [inline Packs/APIVoid/Integrations/APIVoid/APIVoid.py:259]: what type is it?
list or number? 
in the IP one it looks different

File: Packs/APIVoid/Integrations/APIVoid/Pipfile (hunk 1 above)
Comment (yaakovi) [reply Packs/APIVoid/Integrations/APIVoid/Pipfile:30]: it's all the packages in the base docker

File: Packs/APIVoid/Integrations/APIVoid/APIVoid.py (hunk 3 above)
Comment (yaakovi) [reply Packs/APIVoid/Integrations/APIVoid/APIVoid.py:259]: the default value at the end should be an empty list.
will fix

**Pull request merged**